backend/capellacollab/core/authentication/injectables.py

```diff
@@ -59,32 +77,23 @@ class OpenAPIPersonalAccessToken(OpenAPIFakeBase):
     __hash__ = OpenAPIFakeBase.__hash__
 
 
-@dataclasses.dataclass()
-class OpenAPIBearerToken(OpenAPIFakeBase):
-    """Displays the JWT Bearer token as authentication method in the OpenAPI docs"""
-
-    model = openapi_models.HTTPBase(
-        scheme="bearer",
-    )
-    scheme_name = "JWTBearer"
-
-    __hash__ = OpenAPIFakeBase.__hash__
-
-
 async def get_username(
     request: fastapi.Request,
+    oidc_config: oidc_provider.AbstractOIDCProviderConfig = fastapi.Depends(
+        get_oidc_config
+    ),
```

Comment:
It doesn't make sense to fetch the OIDC configuration for unauthenticated routes. I've therefore remove the dependency injection and moved the initialization to JWTAPIKeyCookie.